src/Wizard.js

```diff
@@ -60,19 +66,37 @@ const Wizard = ({ children, onComplete, debug }: Props) => {
     } else {
       nextAction();
     }
+
+    if (state.mode) {
```

Comment:
By referring to `state.mode`, you're inferring an implementation detail in `StateManager`. You don't need that, I think. What about rather using the local data:

```suggestion
    const { stateManager } = this.props;
    if (stateManager) {
```

Reply:
Agreed, fixed in 9adb3e7